pkg/apis/scheduling/v1alpha1/conversion.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/api/scheduling/v1alpha1"
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/kubernetes/pkg/apis/scheduling"
+	"strconv"
+)
+
+func Convert_scheduling_PriorityClass_To_v1alpha1_PriorityClass(in *scheduling.PriorityClass, out *v1alpha1.PriorityClass, s conversion.Scope) error {
+	if err := autoConvert_scheduling_PriorityClass_To_v1alpha1_PriorityClass(in, out, s); err != nil {
+		return err
+	}
+	out.ObjectMeta.Annotations = deepCopyStringMap(out.Annotations)
+
+	if in.Preempting != nil {
+		if out.ObjectMeta.Annotations == nil {
```

Comment:
Thanks @bsalamat  Because we need to keep the RoundTripTypes is ok. If we don't use annotation here.  `scheduling_PriorityClass(v1)` convert to `v1alpha1`, then `v1alpha1` convert to `scheduling_PriorityClass(v1)`. The data`Preempting ` will be lost in the process of transformation. This kind of writing refers to autoscale and deployment. 
https://github.com/kubernetes/kubernetes/blob/1d441c1f93e7cc44f8a200df28a2c8a4bee3a2bb/pkg/apis/autoscaling/v1/conversion.go#L337-L359
https://github.com/kubernetes/kubernetes/blob/1d441c1f93e7cc44f8a200df28a2c8a4bee3a2bb/pkg/apis/apps/v1beta2/conversion.go#L441-L465

Reply:
> The dataPreempting will be lost in the process of transformation. This kind of writing refers to autoscale and deployment.

Add the field to the v1alpha1 type

Reply:
@bsalamat @liggitt  I have added the field to the v1alpha1 type and the v1beta1 type. Delete an annotation. Ready for review~ thanks